src/category_theory/limits/preserves/shapes/biproducts.lean

```diff
@@ -299,6 +390,29 @@ def preserves_biproduct_of_preserves_product {f : J → C} [preserves_limit (dis
       (F.map_cone b.to_cone)).symm (is_limit_of_preserves F hb.is_limit)) $
       cones.ext (iso.refl _) (by tidy) }
 
+/-- If the (product-like) biproduct comparison for `F` and `f` is an isomorphism, then `F`
+    preserves the biproduct of `f`. -/
+def preserves_biproduct_of_is_iso_biproduct_comparison {f : J → C} [has_biproduct f]
+  [has_biproduct (F.obj ∘ f)] [is_iso (biproduct_comparison F f)] : preserves_biproduct f F :=
```

Comment:
Do we have the converse of this? If not, can we have it as a TODO?

Reply:
Yes, the converse already exists under the name `map_biprod`. I have added a comment pointing to this.